src/vs/editor/contrib/inlineCompletions/browser/ghostTextController.ts

```diff
@@ -258,7 +290,11 @@ export class AcceptNextWordOfInlineCompletion extends EditorAction {
 			id: 'editor.action.inlineSuggest.acceptNextWord',
 			label: nls.localize('action.inlineSuggest.acceptNextWord', "Accept Next Word Of Inline Suggestion"),
 			alias: 'Accept Next Word Of Inline Suggestion',
-			precondition: EditorContextKeys.writable
+			precondition: ContextKeyExpr.and(EditorContextKeys.writable, GhostTextController.inlineSuggestionVisible),
+			kbOpts: {
+				weight: 1000,
```

Comment:
You should use `KeybindingWeight` here, and if you need it higher than other editor contribs you can use `KeybindingWeight.EditorContrib + 1`. The way it is now would make it difficult to override the default behavior with an extension or user keybinding.